src/stores/EditSecretUIStore.js

```diff
@@ -0,0 +1,75 @@
+import { Record } from 'immutable';
+import alt from 'utils/alt';
+import makeImmutable from 'utils/makeImmutable';
+
+import EditSecretUIActions from 'actions/EditSecretUIActions';
+import ShowSecretUIActions from 'actions/ShowSecretUIActions';
+import MetadataActions from 'actions/MetadataActions';
+
+const EditSecretUIState = new Record({
+  isEditing: false,
+  data: null,
+});
+
+const {
+  SHOW_SECRET_SUCCESS,
+  HIDE_MODAL,
+}
+= ShowSecretUIActions;
```

Comment:
and that ? https://github.com/secretin/secretin-app/blob/695c9927fbed4c3649f6a859094203a3f2b4d3b7/src/stores/ShowSecretUIStore.js#L27